iothub_client/inc/iothub_device_client_ll.h

```diff
@@ -374,6 +374,24 @@ typedef struct IOTHUB_CLIENT_CORE_LL_HANDLE_DATA_TAG* IOTHUB_DEVICE_CLIENT_LL_HA
 
 #endif /*DONT_USE_UPLOADTOBLOB*/
 
+    /**
+    * @brief    This API sends an acknowledgement to Azure IoT Hub that a cloud-to-device message has been received.
```

Comment:
Per comments about need to call this for freee, I'd put this into the @brief because it's a critical part of what this function is doing.

`This API sends an acknowledgement to Azure IoT Hub that a cloud-to-device message has been received and frees resources associated with the message` 
?